infra/recipes/terraform/azure/k8s_variables.tf

```diff
@@ -0,0 +1,15 @@
+variable "k8s_client_certificate" {
```

Comment:
Are these k8s cluster specific top level variables for Azure? That's why separate file and different from the k8s module?

Reply:
Wondering of this can be part of top level variables.tf if they are same.

Reply:
looks like sandbox variables.tf is empty

Reply:
Yea that makes sense. I added to the top level variables.tf. These variables are to configure the kubernetes provider which is defined on the top level. And then the sandbox variables.tf is empty but still needed in case one wants to add variables in the future.

Reply:
Actually I think we need the k8s_variables file, because when we symlink the global variables.tf, we dont want the k8s variables in every module. So we keep the k8s variables separate so they can be reused but only in certain modules.

Reply:
For example we want to be able to reuse the k8s provider (and therefore variables) across environments. But we don't necessarily want to include those variables in the modules. So I created a new k8s_variables specific file that we can just symlink into the environments that need the k8s provider.